src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -332,7 +332,7 @@ updateUserProfile profile = setProfile profile >> getUserProfile
 
 getWallet :: WalletWebMode ssc m => CAddress -> m CWallet
 getWallet cAddr = do
-    balance <- getBalance =<< decodeCAddressOrFail cAddr
+    balance <- fmap mkCCoin . getBalance =<< decodeCAddressOrFail cAddr
     meta <- getWalletMeta cAddr >>= maybe noWallet pure
     pure $ CWallet cAddr balance meta
   where
```

Comment:
I can introduce them, but this should work as https://github.com/input-output-hk/cardano-sl/blob/master/src/Pos/Wallet/Web/Server/Methods.hs#L591:
```
instance FromHttpApiData Coin where
parseUrlPiece = fmap mkCoin . parseUrlPiece
```
will take info from url: `api/send/address1/address2/10` and it will parse 10 into `Coin 10`. So nothing has to be changed on backend side for it. 

Another way to be more consistent is to change https://github.com/input-output-hk/cardano-sl/blob/master/src/Pos/Wallet/Web/Api.hs#L109 and use `CCoin` here, to change https://github.com/input-output-hk/cardano-sl/blob/master/src/Pos/Wallet/Web/Server/Methods.hs#L591 into: 
```
instance FromHttpApiData CCoin where
parseUrlPiece = fmap CCoin . parseUrlPiece
```

and to use parse CCoin -> Coin with:
```
fromCCoin :: CCoin -> Either Text Coin
fromCCoin = either T.pack mkCoin . raedEitherSafe "not a number" . getCoin
```
in `send` https://github.com/input-output-hk/cardano-sl/blob/master/src/Pos/Wallet/Web/Server/Methods.hs#L351 and `sendExtend` https://github.com/input-output-hk/cardano-sl/blob/master/src/Pos/Wallet/Web/Server/Methods.hs#L356 . We should also, in this case, manually handle and rise exception to handle `Either` (which is done now by `parseUrlPiece`